### DISCUSSION TITLE
Spectrum 2

### DISCUSSION THREAD

**Discussion by jvercaemer** [adobe/react-spectrum#5918] [General]
HI team,

First of all, I would like to thank you so much for your work that help a lot the community. You guys rock !

With the recent announcement of [Spectrum 2](https://s2.spectrum.adobe.com/), I just want to know if we can expect a new react based component library implementing the Spectrum 2 design system and if so, in what timeline ?

I think the Spectrum 2 design system is a major upgrade and I really hope it will possible for the community to use it with the help of one of your UI libraries.

Have a good day !

**Comment by dannify** [comment]
Hi @jvercaemer 
Yes, we will be working on a Spectrum 2 update but as of now we do not have a timeline. Announcements will happen on the website and release notes as we make progress. Stay tuned!
